packages/test/test-gc-sweep-tests/src/test/gcSweepTest.spec.ts

```diff
@@ -74,12 +74,14 @@ describeNoCompat("GC Sweep tests", (getTestObjectProvider) => {
         runtimeOptions,
     );
 
-    const sessionExpiryDurationMs = 3000; // 3 seconds
+    // const sessionExpiryDurationMs = 3000; // 3 seconds
+    const inactiveTimeoutMs = 3000; // 3 seconds
```

Comment:
Shouldn't we make this configurable via test configs or something?

Reply:
Why do we need a default value here?

Reply:
Yeah, there is no current configurable options, everything is hard coded until we add that functionality.

Reply:
I'll add this to the next PR as this seems like a decent sized chunk to look through.